notifications.bs

```diff
@@ -278,7 +278,9 @@ language tag. Validity or well-formedness are not enforced. [[!LANG]]
   </ol>
 
   <li><p>If the notification platform supports sounds, <a>fetch</a>
-  <var>notification</var>'s <a>sound URL</a>, if <a>sound URL</a> is set.
+  <var>notification</var>'s <a>sound URL</a>, if <a>sound URL</a> is set,
+  the <var>notification</var>'s <a>renotify preference flag</a> has been set
+  or the <var>notification</var> is not being replaced.
```

Comment:
Both done, but right now it's still wrong. I defined it as a concept, but I don't think concepts can refer to variables. Do you have an example about how I'd go about defining this as an algorithm?